src/ctap2/mod.rs

```diff
@@ -494,22 +485,7 @@ pub fn sign<Dev: FidoDevice>(
     callback: StateCallback<crate::Result<crate::SignResult>>,
     alive: &dyn Fn() -> bool,
 ) -> bool {
-    if dev.get_protocol() == FidoProtocol::CTAP2 {
-        let info = match dev.get_authenticator_info() {
-            Some(info) => info,
-            None => {
-                callback.call(Err(HIDError::DeviceNotInitialized.into()));
-                return false;
-            }
-        };
-        // Check if extensions have been requested that are not supported by the device
-        if args.extensions.hmac_secret.is_some() && !info.supports_hmac_secret() {
-            callback.call(Err(AuthenticatorError::UnsupportedOption(
-                UnsupportedOption::HmacSecret,
```

Comment:
Seems like `UnsupportedOption::HmacSecret` is unused at this point? Can we remove that, or are we stuck since it once appeared in a public enum?

Reply:
I'll remove it---we're not making any interface guarantees for the 0.4.0-alpha releases.